resources/crds/ripsaw_v1alpha1_stressng_cr.yaml

```diff
@@ -1,15 +1,15 @@
 apiVersion: ripsaw.cloudbulldozer.io/v1alpha1
 kind: Benchmark
 metadata:
-  name: stressng
+  name: stressng-benchmark
   namespace: my-ripsaw
 spec:
   elasticsearch:
-    url: http://my.elasticsearch.server:80
+    url: ""
```

Comment:
All examples are standarized to this URL; we don't should change it

Reply:
Fix it in the following commit:
https://github.com/cloud-bulldozer/benchmark-operator/pull/522/commits/4d5c0fbe895773f08fd8594dc49696d15f8e8eff